aewb_logger/aewb_logger_receiver.c

```diff
@@ -0,0 +1,64 @@
+#include "aewb_logger_receiver.h"
+#include "aewb_logger_types.h"
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <string.h>
```

Comment:
```suggestion
#include <unistd.h>
#include <string.h>
#include <stdlib.h>
#include <stdio.h>

#include "aewb_logger_receiver.h"
#include "aewb_logger_types.h"
```

first are 3rd party libraries, then first party library includes